.gitignore

```diff
@@ -0,0 +1,2 @@
+
```

Comment:
So git ignore only prevents future files from being tracked by git. It does not do anything to files already tracked by the repo. To remove a file from being tracked you can either delete it and commit that delete or `git rm --cached /path/to/file` which deletes from file from git but not your working directory.

Reply:
Cool, thx.